packages/components/src/base-control/style.scss

```
@@ -18,6 +18,9 @@
 	.components-base-control__help {
 		margin-top: -$grid-size;
 		font-style: italic;
-		margin-bottom: 0;
 	}
 }
+
+.components-base-control+.components-base-control {
```

Please add spaces around the `+` for better readability and consistency
```suggestion
.components-base-control + .components-base-control {
```